Documentation/concepts/ipam/index.rst

```diff
@@ -6,35 +6,24 @@
 
 .. _address_management:
 
-******************
-Address Management
-******************
+****************************
+IP Address Management (IPAM)
+****************************
 
-Before we look into the details of Address Management in Cilium, let us
-look at overview of the cilium container networking control flow.
+IP Address Management (IPAM) is responsible for the allocation and management
+of IP addresses used by network endpoints (container and others) managed by
+Cilium. Various address management modes are supported to the meet the needs of
+different users:
```

Comment:
Nit: 
```suggestion
Cilium. Various IPAM modes are supported to the meet the needs of
different users:
```
I find that referring to the topic at hand by its explicit name and proper noun ("IPAM" here) is clearer, than the improper noun, "address management"